### ISSUE TITLE
User Login on Front End

### ISSUE THREAD

**Issue by jakepeyser** [IBM-Bluemix/logistics-wizard#57]
Implement login and session persistence on the front end

Story #11 (Dependent on #50)
Addresses #148

**Comment by benrondeau** [comment]
For clarification sake, this issue will focus on the technical implementation of this task, not the UI.

**Comment by colbycheeze** [comment]
Alright, I have part of this done (very WIP) but putting it on hold until I write the api wrapper and set up some global colors from the design guide so it can come together properly and not be hacky.

![vosorkadru](https://cloud.githubusercontent.com/assets/8884298/17717278/b2cd4382-63d2-11e6-9437-955d4fd48e30.gif)

**Comment by colbycheeze** [comment]
Full login flow was implemented upon creating a demo with #180 
I still need to add a way to login to an existing demo by visiting the correct url.
